default.nix

```diff
@@ -1,9 +1,9 @@
-{ system ? null, allInOne ? true, incremental ? false, img_tag ? "", tag ? "", img_org ? "" }:
+{ system ? null, allInOne ? true, incremental ? false, img_tag ? "", tag ? "", img_org ? "", product_prefix ? "" }:
```

Comment:
We already have img_tag and img_org, so let's use img_prefix here too

Reply:
Yup, makes sense. Changed the name of the variable.